### ISSUE TITLE
Starting desktop build fails. FileNotFoundException non/config.yml

### ISSUE THREAD

**Issue by PaulKlumpp** [deathbeam/yae#19]
Freshly built desktop build fails with this error, using the Hello world project:

```
paul@babidi ~/dev/lala/build/desktop $ java -jar Hello\ World\ v1.0.0.jar 
Exception in thread "main" java.io.FileNotFoundException: non/config.yml (No such file or directory)
        at java.io.FileInputStream.open(Native Method)
        at java.io.FileInputStream.<init>(FileInputStream.java:146)
        at non.hello.desktop.DesktopLauncher.main(DesktopLauncher.java:27)
```

**Comment by deathbeam** [comment]
config.yml is copied from your root project directory to ".non/android/assets/non". Can you check if it is there?

**Comment by PaulKlumpp** [comment]
It is there.

```
paul@dabura ~/dev/emptyy/.non $ find -iname config.yml
./android/assets/non/config.yml
./android/build/intermediates/assets/release/non/config.yml
./hello/config.yml
```

... But I was referring to the desktop version. Does the android config.yml have anything to do with the desktop config.yml?

The desktop jar does have the config.yml inside. See

```
paul@dabura ~/dev/emptyy/build/desktop $ unzip -vt Hello\ World\ v1.0.0.jar  | grep -i config.yml
    testing:     testing: non/config.yml           OK
```

**Comment by deathbeam** [comment]
Because of cross-platform builds compatibility, engine is storing all user data in android/assets temporary subfolder (even on iOS and Desktop builds) because for Android builds, it's asset folder location is fixed, but for desktop and iOS builds it can be changed, so to unify it I changed them to that android folder to prevent double and triple copying.

**Comment by deathbeam** [comment]
Is this issue still happening for you with latest development version?

**Comment by deathbeam** [comment]
Fixed this with commit 96d8064adc81d07575bdbc34a612d6a1d801e9bd
